packages/store/addon/-private/system/model/internal-model.ts

```diff
@@ -1240,20 +1258,48 @@ export default class InternalModel {
     @method adapterDidInvalidate
     @private
   */
-  adapterDidInvalidate(errors) {
-    let attribute;
+  adapterDidInvalidate(parsedErrors, error) {
+    if (false) {
```

Comment:
should be `if (true)` to preserve original behavior for now until FF infra

Reply:
Nah, I'll replace `if (true)` with `if (feature_flag)` before merging. FF infra for this already landed